README.md

```diff
@@ -454,8 +454,6 @@ if the primitives from *random* module are used.
 
 - `generator.nearray(gen: generator a): generator (array a)`
 
-- `generator.dict(gen: generator a): generator (dict a)`
```

Comment:
Hmm, it's unfortunate this def goes away. Could you leave the comment in place in `generator.js` so this is still documented?